metpy/calc/basic.py

```diff
@@ -220,3 +222,28 @@ def heat_index(temperature, rh, mask_undefined=True):
             hi = units.Quantity(masked_array(hi, mask=mask), hi.units)
 
     return hi
+
+
+@exporter.export
+def pressure_to_height_std(pressure):
+    '''Convert pressure data to heights using the U.S. standard atmosphere.
+
```

Comment:
It might be looking for a longer description between the first one-line description docstring and the `Parameters` section. I'd add a section like in `heat_index()` above:
`The implementation uses the formula outlined in [7].`